qemu/tests/cfg/trim_support_test.cfg

```diff
@@ -37,4 +38,15 @@
              filter_options = "<QueryList><Query Id='0' Path='System'><Select Path='System'>"
              filter_options += "*[System[(Level=2) and (EventID=${event_id})]]</Select></Query></QueryList>"
              query_cmd = 'wevtutil qe system /q:"%s" /f:text'
-
+         - with_discard_granularity:
+             guest_trim_cmd = "defrag.exe %s: /u /v /h /o"
+             variants:
+                  - 4K:
```

Comment:
When we don't specify discard_granularity the driver 
adjust this parameter to 4KB automatically.
so it seems to be a duplicated case with  trim_support_test.default.

Reply:
oh, sorry  , one variant is 'guest_trim_cmd ' , so please ignore it, thanks